src/menu.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const React = require('react');
+const mdl = require('material-design-lite/material.min');
+const classnames = require('classnames');
+
+class Menu extends React.Component {
+
+  render(){
+    const {
+      children
+    } = this.props;
+
```

Comment:
this component is going to need to generate a unique `id` and pass it to the components. The render method is going to need to `React.cloneElement` on the children and pass the `id` as either `id` or `htmlFor` props.